src/main/java/com/hashedin/redmask/service/SnowflakeMaskingService.java

```diff
@@ -95,7 +104,67 @@ public void generateSqlQueryForMasking() {
   @Override
   public void executeSqlQueryForMasking() {
     if (!dryRunEnabled) {
-      log.info("Executing script in order to create view in the database.");
+      log.trace("Invoking Redshift masking service to run data masking sql script.");
+      Properties connectionProps = new Properties();
+      connectionProps.setProperty("user", config.getSuperUser());
+      connectionProps.setProperty("password", config.getSuperUserPassword());
+      connectionProps.setProperty("db", config.getDatabase());
+      connectionProps.setProperty("schema", SNOWFLAKE_DEFAULT_SCHEMA);
+      try {
+        executeSqlScript(url, connectionProps, tempFilePath);
+      } catch (IOException ex) {
+        throw new RedmaskRuntimeException(ex);
+      }
+    }
+  }
+
+  @Override
+  protected void executeSqlScript(String url, Properties props,
+                                  File scriptFilePath) throws IOException {
+    Reader reader = null;
+    try (Connection CONN = DriverManager.getConnection(url, props)) {
+      //Initialize the script runner
+      ScriptRunner sr = new ScriptRunner(CONN);
+
+      //Creating a reader object and running the script
+      reader = new BufferedReader(new FileReader(scriptFilePath));
+      sr.setSendFullScript(false);
```

Comment:
Can you add a comment here about why `sr.setSendFullScript(false);`

Reply:
done